client/enduser/src/main/resources/META-INF/resources/app/js/app.js

```diff
@@ -27,394 +29,409 @@ angular.module('info', []);
 
 // Declare app level module which depends on views, and components
 var app = angular.module('SyncopeEnduserApp', [
-  'ui.router',
-  'ui.bootstrap',
-  'ui.select',
-  'ngSanitize',
-  'ngAnimate',
-  'ngResource',
-  'ngCookies',
-  'treasure-overlay-spinner',
-  'ngPasswordStrength',
-  'kendo.directives',
-  'home',
-  'login',
-  'language',
-  'self',
-  'info'
+    'ui.router',
```

Comment:
Please avoid this formatting-only modifications, they make harder to recognize actual changes.